src/marlin_stubs/host/M115.cpp

```diff
@@ -39,18 +40,26 @@ static void cap_line(PGM_P const name, bool ena = false) {
  * M115: Capabilities string
  */
 void GcodeSuite::M115() {
-
+    uint8_t extruders = 0;
+    char extruder[4] = { 0 };
```

Comment:
Wrong variable naming. Some suggestions:
* `extruders` and `extruders_str`
* `extruder_count` and `extruder_count_str`

Reply:
Also, it doesn't make sense to have them declared in the beginning in the file. Declare them just before you use them.
They don't need to be initialized (at elast wewould have a warning if something is wrong.
Ideally, I'd create a dedicated `{}` scope for them and have these variables only in the scope where you're using them (so the `SERIAL_ECHOPGM(extruder);` would be the last thing in the scope).